manifest/com.nalathethird/Fck-SessionCapture/info.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "Fck-SessionCapture",
+	"id": "com.nalathethird.Fck-SessionCapture",
+	"description": "FckSessionCapture allows you to block the automatic upload and/or local generation of session thumbnails ('session captures') in Resonite. This helps protect your privacy by preventing images of your session from being uploaded to the internet or even stored locally, depending on your configuration.",
+	"category": "Technical Tweaks",
+	"sourceLocation": "https://github.com/nalathethird/Fck-SessionCapture",
+	"versions": {
+		"1.0.2": {
+			"releaseUrl": "https://github.com/nalathethird/Fck-SessionCapture/releases/tag/1.0.2",
+			"artifacts": [
+				{
+					"url": "https://github.com/nalathethird/Fck-SessionCapture/releases/download/1.0.2/Fck-SessionCapture.dll",
+					"filename": "Fck-SessionCapture.dll",
+					"installLocation": "/",
```

Comment:
unless this need to be in the root Resonite directory. or somewhere specific, this is not required. For a mod, the location is assumed to be rml_mods already